events/dvg_april_fools.txt

```diff
@@ -0,0 +1,81 @@
+﻿namespace = aprilfools
+
+aprilfools.1 = { #Krakow Indepdence Event
```

Comment:
Everyone should get an even on whats going on. Maybe only krakow has the startup event.